pop/POPAnimatablePropertyInternal.h

```diff
@@ -0,0 +1,23 @@
+//
+//  POPAnimatablePropertyInternal.h
+//  pop
+//
+//  Created by Alexander Cohen on 2015-09-16.
+//  Copyright © 2015 Facebook. All rights reserved.
+//
+
```

Comment:
can we get this header setup to be the same as the other headers? :D

Reply:
Taken care of in https://github.com/facebook/pop/pull/274/commits/f55f93cd07111c206a1c2e2d010377f490ae345f